examples/material-ui/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charSet="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"/>
+  <title>Redux Form</title>
+  <link href="http://redux-form.com/6.0.0-alpha.6/bundle.css"
+    media="screen, projection"
+    rel="stylesheet" type="text/css"/>
+  <link href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.min.css"
+    media="screen, projection" rel="stylesheet" type="text/css"/>
+   <style type="text/css">
+   /*This CSS has been tweked from root site 
+      to match with material-ui*/
+    form > div > div > div {
```

Comment:
@erikras This is the simple override of css in this file. If UI looks good to you we don't need to change anything.